python/dolfinx/wrappers/fem.cpp

```diff
@@ -457,23 +457,26 @@ void fem(py::module& m)
       "locate_dofs_topological",
       [](const std::vector<
              std::reference_wrapper<const dolfinx::fem::FunctionSpace>>& V,
-         const int dim, const Eigen::Ref<const Eigen::ArrayXi>& entities,
+         const int dim, const py::array_t<std::int32_t>& entities,
          bool remote) -> std::array<py::array, 2> {
         if (V.size() != 2)
           throw std::runtime_error("Expected two function spaces.");
         std::array<std::vector<std::int32_t>, 2> dofs
-            = dolfinx::fem::locate_dofs_topological({V[0], V[1]}, dim, entities,
-                                                    remote);
+            = dolfinx::fem::locate_dofs_topological(
+                {V[0], V[1]}, dim, tcb::span(entities.data(), entities.size()),
+                remote);
         return {as_pyarray(std::move(dofs[0])), as_pyarray(std::move(dofs[1]))};
       },
       py::arg("V"), py::arg("dim"), py::arg("entities"),
       py::arg("remote") = true);
   m.def(
       "locate_dofs_topological",
       [](const dolfinx::fem::FunctionSpace& V, const int dim,
-         const Eigen::Ref<const Eigen::ArrayXi>& entities, bool remote) {
-        return as_pyarray(
-            dolfinx::fem::locate_dofs_topological(V, dim, entities, remote));
+         const py::array_t<std::int32_t>& entities, bool remote) {
+        return as_pyarray(dolfinx::fem::locate_dofs_topological(
+            V, dim,
+            tcb::span<const std::int32_t>(entities.data(), entities.size()),
```

Comment:
Use `tcb::span(entities.data(), entities.size())`

Reply:
Was already on the way!